<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_Dialog</class>
 <widget class="QDialog" name="User_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>701</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Bienvenue</string>
    </attribute>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_bienvenue_admin">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Gérer les réservations</string>
    </attribute>
    <widget class="QLabel" name="label_dbState">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>410</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>255</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>255</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>671</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Valid">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>110</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Valid</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Data/Pictures/load.png</normaloff>Data/Pictures/load.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="aerodep">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="aeroarr">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="datedepEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="datearrEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>date dep</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>date arr</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>aeroarr</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>aerodep</string>
     </property>
    </widget>
    <widget class="QPushButton" name="load">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Load all</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Data/Pictures/load.png</normaloff>Data/Pictures/load.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Inscription</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>421</width>
       <height>218</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ID Voyageur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nom et Prénom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nationalité</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Numéro de téléphone</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>E-mail</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Numéro de passport</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Date et heure de reservation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ID Vol</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="txt_idVoyageur">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="txt_nom"/>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_prenom"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_nationalite"/>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_telephone">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_mail"/>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_numPassport"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLineEdit" name="txt_dateReservation">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_heureReservation">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_idVol"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>150</y>
       <width>227</width>
       <height>53</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Data/Pictures/save.png</normaloff>Data/Pictures/save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>save</zorder>
    <zorder>save</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>A propos</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>480</width>
       <height>104</height>
      </rect>
     </property>
     <property name="text">
      <string>Sujet 7 : Application de gestion de réservation du vol
L’objectif est de développer une application graphique permettant de gérer les réservations des vols
au niveau d’une agence de voyage. L’utilisateur de cette application peut :
 Ajouter un vol (numéro vol, date et heure départ, date et heure d’arrivée, Aéroport départ,
aéroport d’arrivée, numéro avion)
 Rechercher et consulter des vols existant par date départ, destination ..., etc.
 Ajouter un voyageur et lui réserver un vol
 Il peut aussi modifier ou annuler un vol.</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_background2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>531</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>530</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>490</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>label_background2</zorder>
  <zorder>tabWidget</zorder>
  <zorder>label_date</zorder>
  <zorder>label_time</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
